Source/Rhetos.Utilities/GeneratedFilesCache.cs

```diff
@@ -93,6 +93,15 @@ public void SaveHash(string sampleSourceFile, byte[] hash)
             File.WriteAllText(hashFile, CsUtility.ByteArrayToHex(hash), Encoding.ASCII);
         }
 
+        public string LoadFromCache(string fileName)

```

Comment:
GeneratedFilesCache class has a different purpose; it is a cache storage for files that are generated in `bin\Generated` folder. The method `LoadFromCache` should be moved to the MacroOrderRepository class, and use `Paths.GeneratedFolder` folder directly instead of GeneratedFilesCacheFolder.

Reply:
The bin\Generated folder is cleared on every deploy so I will have to use the GeneratedCacheFolder

Reply:
OK